### PR TITLE
feat: create divider component

### PR DESCRIPTION
**Pull request by luqven**
## Screenshot
Screen Shot 2022-01-21 at 4 24 38 PM

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #184** 
    * **PR #185** 
      * **PR #193** 
        * **PR #192** 
          * **PR #197** 
            * **PR #194** 
              * **PR #191** 
                * **PR #201** 
                  * **PR #196** 
                    * **PR #198**  (this PR)
                      * **PR #199** 
                        * **PR #200** 
                          * **PR #195** 
      * **PR #187** 
        * **PR #186** 

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/dividers/Divider.module.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import "../../styles/Colors.module.scss";
+
+.divider {
+  background-color: $color_chrome;
+  bottom: 100%;
+  height: 1px;
+  left: 0%;
+  right: 0%;
+  top: -2.04%;
+  width: fill-available;
```
Comment (frederickfogerty) [inline frontend/src/components/dividers/Divider.module.scss:10]: TIL `fill-available` is a thing

File: frontend/src/components/dividers/Divider.module.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@import "../../styles/Colors.module.scss";
+
+.divider {
+  background-color: $color_chrome;
+  bottom: 100%;
+  height: 1px;
+  left: 0%;
+  right: 0%;
+  top: -2.04%;
```
Comment (frederickfogerty) [inline frontend/src/components/dividers/Divider.module.scss:9]: What is the purpose of these percentage values on a 1px high component?

**Review by frederickfogerty — APPROVED** [review]
LGTM

File: frontend/src/components/dividers/Divider.module.scss (hunk 1 above)
Comment (luqven) [reply frontend/src/components/dividers/Divider.module.scss:8]: In fairness its only supported behind a flag in safari and firefox

```
  // fill-available is only supported in Firefox and Safari behind a flag
  width: fill-available;
  width: -webkit-fill-available;
  width: -moz-available;
```

Had to change it to this ☝🏼

File: frontend/src/components/dividers/Divider.module.scss (hunk 2 above)
Comment (luqven) [reply frontend/src/components/dividers/Divider.module.scss]: removed

**Pull request merged**